services/asset_bundle/asset_bundle_impl.cc

```diff
@@ -40,7 +40,8 @@ void AssetBundleImpl::GetAsStream(
   base::FilePath asset_path =
       base::MakeAbsoluteFilePath(asset_dir_->path().Append(asset_string));
 
-  if (!asset_dir_->path().IsParent(asset_path)) {
+  auto asset_dir_abs = base::MakeAbsoluteFilePath(asset_dir_->path());
+  if (!asset_dir_abs.IsParent(asset_path)) {
```

Comment:
We should contribute this patch to Mojo as well because we share this code with them (so that sky_viewer.mojo can use the asset_bundle service built from there).